gordo_components/server/views/base.py

```diff
@@ -233,6 +232,19 @@ def get(self):
         return send_file(buff, attachment_filename="model.tar.gz")
 
 
+class ModelListView(Resource):
+    """
+    List the current models capable of being served by the model
+    """
+
+    @api.doc(description="List the metadata for all models capable of being served.")
+    def get(self, gordo_project: str):
+        collection_dir = os.environ[current_app.config["MODEL_COLLECTION_DIR_ENV_VAR"]]
+        available_models = os.listdir(collection_dir)
+        return jsonify(available_models)
+
+
+api.add_resource(ModelListView, "/gordo/v0/<gordo_project>/models")
```

Comment:
Should we just start putting in the revision here already? E.g.  `/gordo/v0/<gordo_project>/<revision>/models`, where `latest` is a legal revision?

Reply:
I could add that. The PR doesn't have enough changes for you already? :stuck_out_tongue_winking_eye:

Reply:
Actually, perhaps it may be best to forego this into another PR. Adding this will require updates in the tests to a somewhat larger degree. We'll have to train more than one model for different revisions and all that.